add example how to use wildcards in parameter yaml file. (#4950)
ros2/ros2_documentation: source/How-To-Guides/Node-arguments.rst

              some_integers: [1, 2, 3, 4]
              some_doubles : [3.14, 2.718]

>   /**:
>     ros__parameters:
>       wildcard_full: "Full wildcard for any namespaces and any node names"
> 
>   /**/parameter_blackboard:
>     ros__parameters:
>       wildcard_namespace: "Wildcard for any namspaces, but node names"
> 
>   /*:
>     ros__parameters:
>       wildcard_nodename_root_namespace: "Wildcard for any node names, but root namespace"

Reviewer: ```suggestion
  /**:
    ros__parameters:
      wildcard_full: "Full wildcard for any namespaces and any node names"

  /**/parameter_blackboard:
    ros__parameters:
      wildcard_namespace: "Wildcard for a specific node name under any namespace"

  /*:
    ros__parameters:
      wildcard_nodename_root_namespace: "Wildcard for any node names, but only in root namespace"
```

Author: 👍 fixed in https://github.com/ros2/ros2_documentation/pull/4950/commits/e7e00a63522ed0d8c46a185362f7e0c8e1933770

.. note::

>    Wildcards can be used for node names and namespaces

Author: dot missing.

```suggestion
   Wildcards can be used for node names and namespaces.
```

>    Wildcards can be used for node names and namespaces.
>    ``*`` matches a single token delimeted by slashes (``/``).
>    ``**`` matches zero or more tokens delimeted by slashes.
>    Partial matches are not allowed (e.g. ``foo*``).

Reviewer: ```suggestion
   Wildcards can be used for node names and namespaces.
   ``*`` matches a single token delimited by slashes (``/``).
   ``**`` matches zero or more tokens delimited by slashes.
   Partial matches are not allowed (e.g. ``foo*``).
```

Author: 👍 fixed in https://github.com/ros2/ros2_documentation/pull/4950/commits/e7e00a63522ed0d8c46a185362f7e0c8e1933770
